### PR TITLE
chore(prompts): standardize all OHI prompts

### PR DESCRIPTION
**Pull request by jbeveland27**
Updated all OHI prompts to be more succinct.

Also updated Cassandra to use correct port based on #53

### CODE REVIEW

File: recipes/newrelic/cassandra/debian.yml [hunk 1]
```diff
@@ -37,16 +37,16 @@ validationNrql: "SELECT count(*) from CassandraSample where hostname like '{{.HO
...
-    prompt: "Enter the password to connect to Cassandra via JMX."
+    prompt: "Cassandra connection Password (via JMX)"
     secret: true
   - name: "NR_CLI_DB_HOSTNAME"
-    prompt: "Confirm default or enter new value for the Cassandra node hostname."
+    prompt: "Cassandra node Hostname (default: localhost)"
     default: "localhost"
   - name: "NR_CLI_DB_PORT"
-    prompt: "Confirm default port or enter new value. The port where Cassandra is listening and exposing metrics and variables via JMX."
-    default: 7199
```
Comment (Julien4218) [inline recipes/newrelic/cassandra/debian.yml:49]: 7199 should be the correct value. This is also what we document https://docs.newrelic.com/docs/integrations/host-integrations/host-integrations-list/cassandra-monitoring-integration

File: recipes/newrelic/cassandra/rhel.yml [hunk 2]
```diff
@@ -41,16 +41,16 @@ validationNrql: "SELECT count(*) from CassandraSample where hostname like '{{.HO
...
-    prompt: "Enter the password to connect to Cassandra via JMX."
+    prompt: "Cassandra connection Password (via JMX)"
     secret: true
   - name: "NR_CLI_DB_HOSTNAME"
-    prompt: "Confirm default or enter new value for the Cassandra node hostname."
+    prompt: "Cassandra node Hostname (default: localhost)"
     default: "localhost"
   - name: "NR_CLI_DB_PORT"
-    prompt: "Confirm default port or enter new value. The port where Cassandra is listening and exposing metrics and variables via JMX."
-    default: 7199
```
Comment (Julien4218) [inline recipes/newrelic/cassandra/rhel.yml:53]: 7199 should be the correct value. This is also what we document https://docs.newrelic.com/docs/integrations/host-integrations/host-integrations-list/cassandra-monitoring-integration

File: test/deploy/linux/cassandra/install/debian/roles/configure/tasks/main.yml [hunk 3]
```diff
@@ -84,5 +84,5 @@
   - name: Export HOSTNAME
     shell: "echo export NR_CLI_DB_HOSTNAME=localhost >> ~/.bashrc"
   - name: Export DB_PORT
-    shell: "echo export NR_CLI_DB_PORT=7199 >> ~/.bashrc"
```
Comment (Julien4218) [inline test/deploy/linux/cassandra/install/debian/roles/configure/tasks/main.yml:87]: 7199 should be the correct value. This is also what we document https://docs.newrelic.com/docs/integrations/host-integrations/host-integrations-list/cassandra-monitoring-integration

File: test/deploy/linux/cassandra/install/linux2/roles/configure/tasks/main.yml [hunk 4]
```diff
@@ -66,5 +66,5 @@
   - name: Export HOSTNAME
     shell: "echo export NR_CLI_DB_HOSTNAME=localhost >> ~/.bashrc"
   - name: Export DB_PORT
-    shell: "echo export NR_CLI_DB_PORT=7199 >> ~/.bashrc"
```
Comment (Julien4218) [inline test/deploy/linux/cassandra/install/linux2/roles/configure/tasks/main.yml:69]: 7199 should be the correct value. This is also what we document https://docs.newrelic.com/docs/integrations/host-integrations/host-integrations-list/cassandra-monitoring-integration

**Review by Julien4218 — CHANGES_REQUESTED** [review]
I dont believe the cassandra port change is correct.
We should validate all the text changes with an XD person, or rishi, or both.

**Comment by jbeveland27** [comment]
Agreed, changed back to 7199 for Cassandra. Not sure why the issue was written with the other port.

Also agreed on validating the changes with XD. Maybe it's appropriate to merge these changes, and have Emma/Rishi/etc. review from here forward? At the very least this establishes consistency for every recipe.

**Review by Julien4218 — APPROVED** [review]

**Comment by Julien4218** [comment]
@jbeveland27  ✔️

**Pull request merged**